src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -231,6 +236,13 @@ public static void setAuthDisabled(boolean authDisabled) {
         Auth0Connection.authDisabled = authDisabled;
     }
 
+    /**
+     * Override the current {@link #scope} value. This is used principally for setting up test environments.
+     */
+    public static void setScope(String scope) {
+        Auth0Connection.scope = scope;
+    }
```

Comment:
In `TestUtils#getMockHeaders` we set the `Authorization` header with the `auth0UserId` and then extract that in `RequestingUser#createTestUser`. I think it might following the same pattern (with a `scope` header) instead of setting `Auth0Connection#scope` might be a good idea, just so we're keeping all of that code in the same place.

Reply:
Updated.